client/src/config/index.ts

```diff
@@ -13,11 +13,11 @@ export interface AppConfig {
 
 const config: AppConfig = {
     api: {
-        baseUrl: '' //window.ENV_CONFIG.REACT_APP_API_BASE_URL || 'http://localhost:3100'
+        baseUrl: window.ENV_CONFIG.REACT_APP_API_BASE_URL || ''
```

Comment:
Why did you remove this localhost?

Reply:
I assume the 'http://localhost:3100' is only used when the client module project is ran in local, and empty '' can also work, uncomment the environment to adapter other customer scenarios.